docs/core/whats-new/dotnet-9/runtime.md

```diff
@@ -60,6 +60,14 @@ The <xref:System.Runtime.CompilerServices.UnsafeAccessorAttribute> feature allow
 
 :::code language="csharp" source="../snippets/dotnet-9/csharp/UnsafeAccessor.cs":::
 
+## Garbage collection
+
```

Comment:
Are we missing some bits here? Maybe we should comment this heading out if we don't have any updates for it.